src/gui/raster/qgssinglebandpseudocolorrendererwidget.cpp

```diff
@@ -260,6 +351,38 @@ void QgsSingleBandPseudoColorRendererWidget::on_mClassifyButton_clicked()
       }
     }
   }
+  else if ( mClassificationModeComboBox->itemData( mClassificationModeComboBox->currentIndex() ).toInt() == Quantile )
+  { // Quantile
+    mMinMaxWidget->load();
+
+    numberOfEntries = mNumberOfEntriesSpinBox->value();
+
+    int bandNr = mBandComboBox->itemData( bandComboIndex ).toInt();
+    //QgsRasterHistogram myRasterHistogram = mRasterLayer->dataProvider()->histogram( bandNr );
+
+    double myMin = std::numeric_limits<double>::quiet_NaN();
```

Comment:
avoid use of `my` prefixes for variables. They have been confusingly used in very old QGIS code as both local and private member variables. The convention is `mVariable` for member variables, and just `variable` for local variables.